### ISSUE TITLE
Reload from attachment not working with Pentaho reports

### ISSUE THREAD

**Issue by jalzaga** [WilldooIT/Pentaho-reports-for-OpenERP#145]
In Odoo reports there is the possibility to store the report generated as an attached file to the document (e.g. invoice) it gives it a name and you can decide to use that one when you ask to print it instead of generating a new one.
We use the following code to generate the name: (object.state in ('open','paid')) and ('INV'+(object.number or '').replace('/','')+'.pdf') and we mark the reload from attachment option.
The problem is that when the report is a pentaho report, any time you ask to print, a new attachment is generated. It looks as if it doesn´t recognize the name of the existing attachment.

**Comment by richard-willdooit** [comment]
Yes, you are correct.  Reload from attachments has not really been looked at.  Really, Odoo should be checking this flag and doing this before it even considers whether the Pentaho report needs to be called, but obviously not.

This is not likely to be looked at in the short term, sorry.
